tests/test-libraries/libraries/Makefile

```
@@ -56,6 +57,9 @@ endif
 .libs/$(3)/lib$(1).a: .libs/$(3)/lib$(1).o
 	$$(call Q_2,AR     [$(1)]) $(XCODE_DEVELOPER_ROOT)/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar cru $$@ $$<
 
+.libs/$(3)/lib$(1).so: .libs/$(3)/lib$(1).dylib
+	cp .libs/$(3)/lib$(1).dylib .libs/$(3)/lib$(1).so
```

This will:

* Make the build more quiet (Q)
* Clone files instead of copying them on apfs (CP)

```suggestion
	$$(Q) $(CP) .libs/$(3)/lib$(1).dylib .libs/$(3)/lib$(1).so
```